sidecar/config/config.go

```diff
@@ -85,6 +85,8 @@ type Config struct {
 	LogstashServer string `yaml:"logstash_server"`
 
 	LogLevel string `yaml:"log_level"`
+
+	Debug string `yaml:"debug"`
```

Comment:
yes, I agree, since this flag is only for the CLI, it would not make since to define a yaml flag for it since yaml flags are only read once on start up